06-conclusion.Rmd

```diff
@@ -96,20 +100,17 @@ Data visualization allows decision makers to make sense out of the visible patte
 
 * **Deliver value in your visualization.** Many graphs may be aesthetically pleasing or exciting but deliver no value. A graph with good value should deliver important insights to aid decision-making or prompt an action. Keep in mind, the main objective of a visualization is to deliver value, and each design decision should help achieve this.
  
-* **Understanding your audience is the most difficult and important part of developing a visualization.** Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
+* **Understanding your audience is the most difficult and important part of developing a visualization.* Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
 
-* **Share what you have found.** A visualization can be an effective influencer if used in the correct way and at the correct time. 
+* **Share what you have found** A visualization can be an effective influencer if used in the correct way and at the correct time. 
 
 
-* **Think as author / reader** When designing graphs, try to stand at the reader side, to check whether the graph is making sense or if there is any misleading information being showcased. As a reader, one needs to detect if there is any visualization deception in the graph. Before making any conclusion, understanding details and information the graph is providing like legend, caption, numbers, etc. is important. 
+* **Think as author / reader* When designing graphs, try to stand at the reader side, to check whether the graph is making sense or if there is any misleading information being showcased. As a reader, one needs to detect if there is any visualization deception in the graph. Before making any conclusion, understanding details and information the graph is providing like legend, caption, numbers, etc. is important. 
```

Comment:
Hi Prerna, 
I think by mistake one * has been deleted.
The whole paragraph is bold now. Please add ** after the author/reader.

Reply:
Hey. Thanks for pointing this out. will change this now.